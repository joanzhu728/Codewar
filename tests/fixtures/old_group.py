# Deprecated and should not be used
import codewars_test as test

test.describe("group 1")
test.it("test 1")
test.assert_equals(1, 1)
