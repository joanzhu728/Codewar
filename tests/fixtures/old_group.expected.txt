
<DESCRIBE::>group 1

<IT::>test 1

<PASSED::>Test Passed
